extension-client/src/command/shell/terraform/index.ts

```diff
@@ -58,3 +57,60 @@ export async function hcltojsonFunc() {
     const versionsPath = util.workspace.getWorkspaceVersionsFilePath();
     return util.workspace.writeToFile(versionsPath, jsonData);
 }
+
+
+export async function estimateCost(writeEmitter:any,closeEmitter:any): Promise<any> {
+    
+    const newLine = '\r\n';
+    const extraNewline = '\n';
+    const folderName = util.workspace.getWorkspaceName();
+
+    const TERRAFORM_PLAN_COMMAND = `terraform plan --out .vscode-ibmcloud-schematics/${folderName}.binary`;
+    const TERRAFORM_SHOW_COMMAND = `terraform show -json .vscode-ibmcloud-schematics/${folderName}.binary > .vscode-ibmcloud-schematics/${folderName}.json`;
+    const TERRAFORM_API_COMMAND = 'IC_API_KEY=';
+    const TERRAFORM_COST_COMMAND = `tfcost plan .vscode-ibmcloud-schematics/${folderName}.json --json`;
+    try{
+        await util.workspace.createCredentialFile();
+        writeEmitter.fire('\x1b[1m\x1b[33m' + "Running terraform init" + '\x1b[0m' + newLine);
+        await init();
+        writeEmitter.fire('\x1b[1m\x1b[32mSuccess!\x1b[0m ' + "terraform init" + newLine + extraNewline);
+        writeEmitter.fire('\x1b[1m\x1b[33m' + "Running terraform plan" + '\x1b[0m' + newLine);
+        await shell.execute(TERRAFORM_PLAN_COMMAND);
+        writeEmitter.fire('\x1b[1m\x1b[32mSuccess!\x1b[0m ' + "terraform plan" + newLine + extraNewline);
+        writeEmitter.fire('\x1b[1m\x1b[33m' + "Creating cost.json file" + '\x1b[0m' + newLine);
```

Comment:
We have a terminal util [here](https://github.com/IBM-Cloud/vscode-ibmcloud-schematics/blob/main/extension-client/src/util/terminal.ts). Any particular reason this was not used here ?